src/Utils.php

```diff
@@ -131,6 +131,10 @@ public static function uriTemplate($template, array $variables)
      */
     public static function jsonDecode($json, $assoc = false, $depth = 512, $options = 0)
     {
+        if ($json === '' || $json === null) {
```

Comment:
Just for curiosity: this has nothing to do with PHP 7 compatibility, right? AFAIK json_decode would just return null anyway, so this might be a performance gain. I guess HHVM would fail if it's null, do it's compatibility thing there.